controller/Security.php

```diff
@@ -58,4 +75,9 @@ private function getOidcLoginAuthenticator(): OidcLoginAuthenticatorInterface
     {
         return $this->getServiceLocator()->get(OidcLoginAuthenticatorProxy::class);
     }
+
+    private function getAccessTokenGenerator(): AccessTokenGeneratorInterface
```

Comment:
```suggestion
    private function getAccessTokenResponseGenerator(): AccessTokenGeneratorInterface
```
Maybe add response in the name, cause it actually generates the response. Wdyt?

Reply:
I think you should remove `AccessTokenGeneratorInterface`, cause it is not used.